test/OrchardCore.Commerce.Tests.UI/UITestBase.cs

```diff
@@ -34,7 +37,18 @@ protected UITestBase(ITestOutputHelper testOutputHelper)
                     AccessibilityCheckingConfiguration
                         .ConfigureWcag21aa(axeBuilder)
                         .DisableRules("color-contrast");
+                configuration.HtmlValidationConfiguration.AssertHtmlValidationResultAsync =
+                    AssertHtmValidationResultAsync;
 
                 if (changeConfigurationAsync != null) await changeConfigurationAsync(configuration);
             });
+
+    private static async Task AssertHtmValidationResultAsync(HtmlValidationResult validationResult)
+    {
+        // The built-in Orchard Core pager has a rel attribute on the <a> tag which is not allowed by the HTML
+        // specification. We are ignoring this error for all tests, since most of them will display the pager.
+        var errors = (await validationResult.GetErrorsAsync())
+            .Where(error => !error.ContainsOrdinalIgnoreCase("\"rel\" attribute cannot be used in this context"));
+        errors.ShouldBeEmpty();
```

Comment:
You should be able to surround the `@await DisplayAsync(Model.Pager)` in _ProductListPart.cshtml_ with [disable and then enable directives](https://html-validate.org/usage/#inline-configuration) for (I guess) the [attribute-misuse](https://html-validate.org/rules/attribute-misuse.html) rule. That way we only disable this rule for the pager and won't have false positives or needlessly created validation reports.